esmvalcore/_recipe.py

```diff
@@ -426,14 +416,35 @@ def _update_fx_files(step_name, settings, variable, config_user, fx_vars):
     """Update settings with mask fx file list or dict."""
     if not fx_vars:
         return
+    for fx_var, fx_info in fx_vars.items():
+        if not fx_info:
```

Comment:
isn't `fx_info` an empty dict `{}` to the very least at all times?

Reply:
It can be `None` as well, for example when the input is given as a list such as ['areacello', 'areacella']. That is why it's inizialited as a dict.